src/components/cards-grid-list/cards-grid-list.module.css

```diff
@@ -1,28 +1,30 @@
 .listRoot {
-  display: grid;
-  gap: 24px;
-  list-style: none;
-  margin: 0;
-  padding: 0;
+	display: grid;
+
+	/* Note: --grip-gap set in JS */
+	gap: var(--grid-gap);
```

Comment:
Mostly whitespace changes here.

The substantial change is that `--grid-gap` is now set via JS.